openedx/adg/admin_dashboard/admin_settings/models.py

```diff
@@ -1,3 +1,56 @@
 """
 Models for `admin_settings` app
 """
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from openedx.adg.lms.applications.models import BusinessLine
+
+
+class Invitation(models.Model):
+    """
+    Model to save invitations sent out to users to grant them ADG admin or business unit admin roles.
+    """
+
+    email = models.EmailField(verbose_name=_('Email'), max_length=255, )
+    business_unit = models.ForeignKey(
+        BusinessLine,
+        verbose_name=_('Business Unit'),
+        related_name='invitations',
+        on_delete=models.CASCADE,
+        null=True,
+        blank=True,
+    )
+
+    ADG_ADMIN = 'adg_admin'
+    BU_ADMIN = 'bu_admin'
+    VIEWER = 'viewer'
+
+    PERMISSION_CHOICES = (
+        (ADG_ADMIN, _('ADG Admin')),
+        (BU_ADMIN, _('BU Admin')),
+        (VIEWER, _('Viewer'))
+    )
+
+    permission = models.CharField(verbose_name=_('Permission'), choices=PERMISSION_CHOICES, max_length=9, )
+
+    ACTIVE = 'active'
+    PENDING = 'pending'
+
+    STATUS_CHOICES = (
+        (ACTIVE, _('Active')),
+        (PENDING, _('Pending')),
+    )
+
+    status = models.CharField(verbose_name=_('Status'), choices=STATUS_CHOICES, max_length=7, )
+
+    class Meta:
+        app_label = 'admin_settings'
+        indexes = [
+            models.Index(name='email_index', fields=['email']),
+            models.Index(name='status_index', fields=['status']),
+            models.Index(name='permission_index', fields=['permission']),
```

Comment:
We can create indexes by just adding `db_index=True` attribute to any field. Also I don't think we need all these indexes. At least, we do not need index for permission and my POV is that this table will be very small so it won't make much difference as well. WDYT @AliAdnanSohail ?